src/main.go

```diff
@@ -47,9 +47,13 @@ var (
 	retryConnCount   int
 	collationDisable bool
 	checkErr         bool
+	disableSource    bool
 )
 
 func init() {
+	// Disable the `--source` command by default to avoid breaking existing tests
+	disableSource = true
+
```

Comment:
Should there be a program argument to set it enabled by default, similar to `--check-error` argument, so it would work with test files that contain `--source`'d files that does not exists or have not yet been tested successfully?